.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,24 @@
+name: .NET Core Test and Publish
+
+on: pull_request
+
+env:
+  DOTNET_VERSION: 3.0.100
+
+jobs:
+
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Chectout
```

Comment:
```suggestion
    - name: Checkout
```

Reply:
I also noticed this on the NEO, we can fix together with another PR there.